### DISCUSSION TITLE
How can I make the Logger to print date instead timestamp?

### DISCUSSION THREAD

**Discussion by bstsnail** [adonisjs/core#1621] [Help] [answered]
Hi, I'm new to javascript& adonis. Currently the Logger print the timestamp in log, how can I make it to print date format?
![image](https://user-images.githubusercontent.com/16055826/93548925-d90cdc80-f99a-11ea-8573-6300d6063507.png)

**Comment by thetutlage** [comment, marked as answer]
Which version of the framework?

**Reply by bstsnail** [reply]
5.0.0-preview-rc-1.11

**Reply by bstsnail** [reply]
@thetutlage any clue?

**Reply by thetutlage** [reply]
Ohh yeah, you can define a custom timestamp formatter in your config file `config/app.ts` file. This is how it should go.

```ts
import pino from 'pino'

export const logger: LoggerConfig = {
   timestamp: pino.stdTimeFunctions.isoTime,

   // rest of the config
}
```

**Reply by bstsnail** [reply]
Great, thanks a lot, it works

**Reply by bstsnail** [reply]
Hi, @thetutlage 
Can the Logger framework add the line number, function names, ts file name .etc in the logs?

**Reply by thetutlage** [reply]
Nope. That requires instrumentation and logging and instrumentation are not the same thing
